addons/udes_security/models/res_partner.py

```diff
@@ -0,0 +1,14 @@
+from odoo import api, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    @api.multi
+    def write(self, vals):
+        password = self._context.get("password")
```

Comment:
Why `self._context` instead of `self.env.context`?

Reply:
`self._context` is just a shortcut for `self.env.context`, they do the same thing, I think `_context` looks a little cleaner.